### PR TITLE
Feb 20 bugs + Inline editing

### PR DESCRIPTION
**Pull request by anpolimus**
## Steps for review

- [ ] Google translate link on mobile menu duplicates after multiple hamburger click
Attach.
We've updated google translate widget for mobile. Now it should not generate additional links after click.

- [x] #1770 Open Locations page, select items with 2 amenities items. All items should match user request.
- [ ] #1870 At carnation, edit banner paragraph, change of color should appear at page after save.
- [x] #1867 Location page at mobile is now scrollable. Before that user was not able to scroll because of map.
- [ ] Backport. Activity Finder has a bug, when keywords are not clearing after Clear All click.
- [x] Inline Editing. 
   If you enable Geysir module, Inline editing will appear for landing pages.
   Enabled paragraphs to add/edit:
     '1c',
    '2c',
    '3c',
    '4c',
    'banner',
    'faq',
    'gallery',
    'grid_content',
    'promo_card',
    'secondary_description_sidebar',
    'simple_content',
    'story_card',
    'teaser',
    'small_banner', 

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer. See [Code of Conduct](https://github.com/ymcatwincities/openy/wiki/Open-Y-Code-of-Conduct-and-Best-Practices).
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

**Comment by bjwilkins** [comment]
Can one of the admins verify this patch? Use "o+k to test" or ''t+est this please" for manual build execution.

**Comment by podarok** [comment]
ok to test

File: modules/openy_features/openy_block/modules/openy_block_branch_contacts_info/src/Plugin/Block/BranchContactsInfo.php [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+   * {@inheritdoc}
+   */
+  public function build() {
+    $render_array = [];
+
+    $node = $this->routeMatch->getParameter('node');
+    if ($node instanceof NodeInterface && $node->getType() == 'branch') {
+      $render_array = ['#theme' => 'block_branch_contacts_info'];
+
+      $address_array = $node->get('field_location_address')->get(0)->toArray();
```
Comment (podarok) [inline modules/openy_features/openy_block/modules/openy_block_branch_contacts_info/src/Plugin/Block/BranchContactsInfo.php]: what if `get(0)` returns null?
Could you add verification here, please

File: modules/openy_features/openy_block/modules/openy_block_branch_contacts_info/src/Plugin/Block/BranchContactsInfo.php [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+    return $render_array;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCacheTags() {
+    if ($node = $this->routeMatch->getParameter('node')) {
+      //if there is node add its cachetag
```
Comment (podarok) [inline modules/openy_features/openy_block/modules/openy_block_branch_contacts_info/src/Plugin/Block/BranchContactsInfo.php]: ensure you have Drupal style comments settings in IDE.
Should be 
`// If there is a node add its cachetag.`
Not blocking release

File: modules/openy_features/openy_block/modules/openy_block_branch_contacts_info/templates/block-branch-contacts-info.html.twig [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<div class="menu-spacer"></div>
```
Comment (podarok) [inline modules/openy_features/openy_block/modules/openy_block_branch_contacts_info/templates/block-branch-contacts-info.html.twig:17]: add template docblock, please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2010/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2010/

File: patches/geysir/add-reload-page.patch [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ use Drupal\Core\Ajax\HtmlCommand;
++use Drupal\Core\Ajax\InvokeCommand;
+ use Drupal\Core\Form\FormStateInterface;
+ use Drupal\geysir\Ajax\GeysirCloseModalDialogCommand;
+ 
+@@ -92,6 +93,7 @@ class GeysirModalParagraphForm extends GeysirParagraphForm {
+       );
+ 
+       $response->addCommand(new GeysirCloseModalDialogCommand());
++      $response->addCommand(new InvokeCommand(NULL, 'reloadPageAjaxAction'));
```
Comment (podarok) [inline patches/geysir/add-reload-page.patch:33]: Is this patch added to the respective project as an issue? I'd like us to ensure we are not storing patches, but trying to fix them in external dependencies

File: themes/openy_themes/openy_carnation/dist/img/geysir/small-banner.svg [hunk 5]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (podarok) [inline themes/openy_themes/openy_carnation/dist/img/geysir/small-banner.svg]: Should these files be on global, not on carnation only level?
somewhere like here https://github.com/ymcatwincities/openy/tree/8.x-2.x/themes/openy_themes/scss or in ```../img/``` ?

File: themes/openy_themes/openy_lily/css/style.css.map [hunk 6]
```diff
@@ -1,7 +1,7 @@
...
...,IAAI;EACf,MAAM,EAAE,IAAI;EACZ,WAAW,EAAE,IAAI;EACjB,cAAc,EAAE,SAAS;EACzB,WAAW,EAAE,MAAM;EACnB,OAAO,EAAE,MAAM;EACf,MAAM,EAAE,YAAY;;A1BhKxB,wBAAwB;E0BoJtB,gEAAQ;IAcJ,MAAM,EAAE,cAAc;IACtB,KAAK,E9BxNH,OAAO;;;A8B0NX,wEAAQ;EACN,WAAW,EAAE,IAAI;EACjB,WAAW,EAAE,IAAI;;A1BvKvB,wBAAwB;E0BqKpB,wEAAQ;IAIJ,WAAW,EAAE,IAAI;;;AAEnB,4EAAI;EACF,SAAS,EAAE,IAAI;EACf,KAAK,EnB9FV,OAAO;;AP/EV,wBAAwB;E0B2KlB,4EAAI;IAIA,SAAS,EAAE,IAAI;;;AASzB,8GAAyB;EACvB,KAAK,E9B9OC,OAAO;E8B+Ob,UAAU,E9BnPD,OAAO;;;A8ByPpB,0CAAS;EACP,aAAa,EAAE,IAAI;;AACnB,6CAAG;EACD,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,IAAI;EACjB,aAAa,EAAE,IAAI;EACnB,WAAW,EAAE,GAAG;;A1BtMlB,wBAAwB;E0BkMxB,6CAAG;IAOC,SAAS,EAAE,IAAI;IACf,WAAW,EAAE,IAAI;;;AAEnB,gDAAG;EACD,UAAU,EAAE,MAAM;;AAItB,sDAAY;E7B5Qd,WAAW,EAAE,qCAAqC;E6B8Q9C,KAAK,EnBpMF,OAAO;EmBqMV,SAAS,EAAE,IAAI;EACf,aAAa,EAAE,IAAI;EACnB,UAAU,EAAE,CAAC;EACb,WAAW,EAAE,OAAO;EACpB,WAAW,EAAE,MAAM;;A1BxNrB,wBAAwB;E0BiNxB,sDAAY;IASR,SAAS,EAAE,IAAI;;;AAGnB,0EAAgC;EAC9B,MAAM,EAAE,QAAQ;;A1B9NlB,wBAAwB;E0B6NxB,0EAAgC;IAG5B,MAAM,EAAE,WAAW;;;A1BhOvB,2BAAwB;E0B6NxB,0EAAgC;IAM5B,YAAY,EAAE,CAAC;;;AAEjB,uFAAa;EACX,OAAO,EAAE,WAAW;EACpB,OAAO,EAAE,YAAY;EACrB,OAAO,EAAE,WAAW;EACpB,OAAO,EAAE,IAAI;EACb,SAAS,EAAE,IAAI;;AAEjB,uFAAa;EACX
```
Comment (podarok) [inline themes/openy_themes/openy_lily/css/style.css.map:4]: we can remove map from production. No need for it, it's development feature

File: themes/openy_themes/openy_lily/img/geysir/alert.svg [hunk 7]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (podarok) [inline themes/openy_themes/openy_lily/img/geysir/alert.svg]: definitely better to keep it on a global level only once.

File: themes/openy_themes/openy_lily/sass/modules/_geysir.scss [hunk 8]
```diff
@@ -0,0 +1,71 @@
+// Modal window styles.
```
Comment (podarok) [inline themes/openy_themes/openy_lily/sass/modules/_geysir.scss:1]: if the file is same for all 3 themes, let's keep it on global level via include, like we do for https://github.com/ymcatwincities/openy/blob/8.x-2.x/themes/openy_themes/scss/modules.scss

File: themes/openy_themes/openy_lily/sass/style.scss [hunk 9]
```diff
@@ -59,6 +59,7 @@
 @import 'modules/repeating-schedules';
 @import 'modules/gtranslate';
 @import 'modules/home_branch';
+@import 'modules/geysir';
```
Comment (podarok) [inline themes/openy_themes/openy_lily/sass/style.scss:62]: https://github.com/ymcatwincities/openy/blob/8.x-2.x/themes/openy_themes/scss/modules.scss

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2010/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2010/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2010/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2010/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2010/blogWCAG2AAhtmlcs.txt

File: themes/openy_themes/openy_rose/Gemfile.lock [hunk 10]
```diff
@@ -0,0 +1,71 @@
+GEM
```
Comment (podarok) [inline themes/openy_themes/openy_rose/Gemfile.lock:1]: no need for lock file if we added it to .gitignore
Or let's remove it from .gitignore if we should keep the file in repo

**Review by podarok — CHANGES_REQUESTED** [review]
Please, see review concerns below

**Comment by podarok** [comment]
@anpolimus @atiupa @jediandrey could you provide steps for review, please
I see a lot of fixes and we definitely must to test everything

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2010/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2014/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2014/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2014/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2014/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2014/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2014/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2014/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Upgrade Path site check installed at http://upgrade.openy.cibox.tools/build2014/

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2014/

File: themes/openy_themes/openy_lily/sass/modules/_geysir.scss (hunk 8 above)
Comment (jediandrey) [reply themes/openy_themes/openy_lily/sass/modules/_geysir.scss:1]: the same for 95%, but has some differences, compare .ui-dialog-titlebar-close style for carnation and rose for example.

**Comment by gundevel** [comment]
!!! Couldn't read comment file !!!
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2022/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #1890 of commit 7400847684b848a8176a820c459708cd137f3daa automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER_COMPOSER/workspace/build2022/openy-profile # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git2774919241428409991.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1 # timeout=5
 > git config --local --remove-section credential # timeout=10
ERROR: Error cloning remote repo 'origin'
hudson.plugins.git.GitException: Command "git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1" returned status code 128:
stdout: 
stderr: remote: Enumerating objects: 14117, done.
remote: Counting objects:   0% (1/14117)
remote: Counting objects:   1% (142/14117)
remote: Counting objects:   2% (283/14117)
remote: Counting objects:   3% (424/14117)
remote: Counting objects:   4% (565/14117)
remote: Counting objects:   5% (706/14117)
remote: Counting objects:   6% (848/14117)
remote: Counting objects:   7% (989/14117)
remote: Counting objects:   8% (1130/14117)
remote: Counting objects:   9% (1271/14117)
remote: Counting objects:  10% (1412/14117)
remote: Counting objects:  11% (1553/14117)
remote: Counting objects:  12% (1695/14117)
remote: Counting objects:  13% (1836/14117)
remote: Counting objects:  14% (1977/14117)
remote: Counting objects:  15% (2118/14117)
remote: Counting objects:  16% (2259/14117)
remote: Counting objects:  17% (2400/14117)
remote: Counting objects:  18% (2542/14117)
remote: Counting objects:  19% (2683/14117)
remote: Counting objects:  20% (2824/14117)
remote: Counting objects:  21% (2965/14117)
remote: Counting objects:  22% (3106/14117)
remote: Counting objects:  23% (3247/14117)
remote: Counting objects:  24% (3389/14117)
remote: Counting objects:  25% (3530/14117)
remote: Counting objects:  26% (3671/14117)
remote: Counting objects:  27% (3812/14117)
remote: Counting objects:  28% (3953/14117)
remote: Counting objects:  29% (4094/14117)
remote: Counting objects:  30% (4236/14117)
remote: Counting objects:  31% (4377/14117)
remote: Counting objects:  32% (4518/14117)
remote: Counting objects:  33% (4659/14117)
remote: Counting objects:  34% (4800/14117)
remote: Counting objects:  35% (4941/14117)
remote: Counting objects:  36% (5083/14117)
remote: Counting objects:  37% (5224/14117)
remote: Counting objects:  38% (5365/14117)
remote: Counting objects:  39% (5506/14117)
remote: Counting objects:  40% (5647/14117)
remote: Counting objects:  41% (5788/14117)
remote: Counting objects:  42% (5930/14117)
remote: Counting objects:  43% (6071/14117)
remote: Counting objects:  44% (6212/14117)
remote: Counting objects:  45% (6353/14117)
remote: Counting objects:  46% (6494/14117)
remote: Counting objects:  47% (6635/14117)
remote: Counting objects:  48% (6777/14117)
remote: Counting objects:  49% (6918/14117)
remote: Counting objects:  50% (7059/14117)
remote: Counting objects:  51% (7200/14117)
remote: Counting objects:  52% (7341/14117)
remote: Counting objects:  53% (7483/14117)
remote: Counting objects:  54% (7624/14117)
remote: Counting objects:  55% (7765/14117)
remote: Counting objects:  56% (7906/14117)
remote: Counting objects:  57% (8047/14117)
remote: Counting objects:  58% (8188/14117)
remote: Counting objects:  59% (8330/14117)
remote: Counting objects:  60% (8471/14117)
remote: Counting objects:  61% (8612/14117)
remote: Counting objects:  62% (8753/14117)
remote: Counting objects:  63% (8894/14117)
remote: Counting objects:  64% (9035/14117)
remote: Counting objects:  65% (9177/14117)
remote: Counting objects:  66% (9318/14117)
remote: Counting objects:  67% (9459/14117)
remote: Counting objects:  68% (9600/14117)
remote: Counting objects:  69% (9741/14117)
remote: Counting objects:  70% (9882/14117)
remote: Counting objects:  71% (10024/14117)
remote: Counting objects:  72% (10165/14117)
remote: Counting objects:  73% (10306/14117)
remote: Counting objects:  74% (10447/14117)
remote: Counting objects:  75% (10588/14117)
remote: Counting objects:  76% (10729/14117)
remote: Counting objects:  77% (10871/14117)
remote: Counting objects:  78% (11012/14117)
remote: Counting objects:  79% (11153/14117)
remote: Counting objects:  80% (11294/14117)
remote: Counting objects:  81% (11435/14117)
remote: Counting objects:  82% (11576/14117)
remote: Counting objects:  83% (11718/14117)
remote: Counting objects:  84% (11859/14117)
remote: Counting objects:  85% (12000/14117)
remote: Counting objects:  86% (12141/14117)
remote: Counting objects:  87% (12282/14117)
remote: Counting objects:  88% (12423/14117)
remote: Counting objects:  89% (12565/14117)
remote: Counting objects:  90% (12706/14117)
remote: Counting objects:  91% (12847/14117)
remote: Counting objects:  92% (12988/14117)
remote: Counting objects:  93% (13129/14117)
remote: Counting objects:  94% (13270/14117)
remote: Counting objects:  95% (13412/14117)
remote: Counting objects:  96% (13553/14117)
remote: Counting objects:  97% (13694/14117)
remote: Counting objects:  98% (13835/14117)
remote: Counting objects:  99% (13976/14117)
remote: Counting objects: 100% (14117/14117)
remote: Counting objects: 100% (14117/14117), done.
remote: Compressing objects:   0% (1/7866)
remote: Compressing objects:   1% (79/7866)
remote: Compressing objects:   2% (158/7866)
remote: Compressing objects:   3% (236/7866)
remote: Compressing objects:   4% (315/7866)
remote: Compressing objects:   5% (394/7866)
remote: Compressing objects:   6% (472/7866)
remote: Compressing objects:   7% (551/7866)
remote: Compressing objects:   8% (630/7866)
remote: Compressing objects:   9% (708/7866)
remote: Compressing objects:  10% (787/7866)
remote: Compressing objects:  11% (866/7866)
remote: Compressing objects:  12% (944/7866)
remote: Compressing objects:  13% (1023/7866)
remote: Compressing objects:  14% (1102/7866)
remote: Compressing objects:  15% (1180/7866)
remote: Compressing objects:  16% (1259/7866)
remote: Compressing objects:  17% (1338/7866)
remote: Compressing objects:  18% (1416/7866)
remote: Compressing objects:  19% (1495/7866)
remote: Compressing objects:  20% (1574/7866)
remote: Compressing objects:  21% (1652/7866)
remote: Compressing objects:  22% (1731/7866)
remote: Compressing objects:  23% (1810/7866)
remote: Compressing objects:  24% (1888/7866)
remote: Compressing objects:  25% (1967/7866)
remote: Compressing objects:  26% (2046/7866)
remote: Compressing objects:  27% (2124/7866)
remote: Compressing objects:  28% (2203/7866)
remote: Compressing objects:  29% (2282/7866)
remote: Compressing objects:  30% (2360/7866)
remote: Compressing objects:  31% (2439/7866)
remote: Compressing objects:  32% (2518/7866)
remote: Compressing objects:  33% (2596/7866)
remote: Compressing objects:  34% (2675/7866)
remote: Compressing objects:  35% (2754/7866)
remote: Compressing objects:  36% (2832/7866)
remote: Compressing objects:  37% (2911/7866)
remote: Compressing objects:  38% (2990/7866)
remote: Compressing objects:  39% (3068/7866)
remote: Compressing objects:  40% (3147/7866)
remote: Compressing objects:  41% (3226/7866)
remote: Compressing objects:  42% (3304/7866)
remote: Compressing objects:  43% (3383/7866)
remote: Compressing objects:  44% (3462/7866)
remote: Compressing objects:  45% (3540/7866)
remote: Compressing objects:  46% (3619/7866)
remote: Compressing objects:  47% (3698/7866)
remote: Compressing objects:  48% (3776/7866)
remote: Compressing objects:  49% (3855/7866)
remote: Compressing objects:  50% (3933/7866)
remote: Compressing objects:  51% (4012/78
```

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2026/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2026/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2026/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2026/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2026/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2026/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2026/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2026/

**Comment by bjwilkins** [comment]
Build comment file: 
--------------

Check Open Y Installation Wizard at http://install.openy.cibox.tools/build2028/install.php

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy.cibox.tools:8080/job/PR_BUILDER_COMPOSER/2028/

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://openy.cibox.tools/build2028/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://openy.cibox.tools/build2028/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://openy.cibox.tools/build2028/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://openy.cibox.tools/build2028/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://openy.cibox.tools/build2028/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Open Y Complete profile installed at http://openy.cibox.tools/build2028/

**Comment by podarok** [comment]
@anpolimus I'm ready to merge. Any concerns or leftovers?

**Pull request merged**